runhouse/constants.py

```diff
@@ -62,3 +63,7 @@
 # cluster status constants
 DOUBLE_SPACE_UNICODE = "\u00A0\u00A0"
 BULLET_UNICODE = "\u2022"
+DEFAULT_POST_STATUS_TO_DEN_INTERVAL_SIZE = 120
+# post status to den thread will start running 2 min after cluster servlet setup.
+# This delay enables the cluster setup to finish before running the post status job,
+DELAY_STATUS_POST_THREAD = 120
```

Comment:
name was confusing to me at first - maybe something more general like `STATUS_CHECK_DELAY`